app/controllers/application_controller.rb

```diff
@@ -1,9 +1,8 @@
 class ApplicationController < ActionController::Base
-  before_action :authenticate
+  # before_action :authenticate
```

Comment:
Is there a reason this is disabled?  We need this (for now) on heroku so that it doesn't accidentally get spidered by google.